docs/variants/msi_z690/heads.md

```diff
@@ -0,0 +1,130 @@
+
+Heads-based variant of Dasharo firmware compatible with MSI PRO Z690-A boards
+is offered as a Technology Preview Release.
+
+Please consider to support the project financially by purchasing the
+[Dasharo Entry Subscription for Desktop](https://shop.3mdeb.com/product-category/dasharo-entry-subscription/).
+With this subscription, you will get access to the
+[Transition from Dasharo UEFI to Heads variant](#transition-from-dasharo-uefi-to-heads-variant)
+feature in Dasharo Tools Suite and support from Dasharo directly via Matrix.
+The subscription is not automatically renewed.
+
+## Releases
+
+Following Release Notes describe status of Dasharo (coreboot + Heads) firmware
+development compatible with MSI PRO Z690-A boards.
+
+COMING SOON
+
+## Hardware Configuration Matrix
+
+Generally, the same [hardware configuration](hardware-matrix.md) as for the
+UEFI variant applies.
+
+A notable addition is usage of the
+[Nitrokey 3A Mini](https://shop.nitrokey.com/shop/nk3am-nitrokey-3a-mini-149)
+USB device, which is required for Heads installation and usage.
+
+## Test Matrix
+
+Please refer to the [tests results spreadsheet](TBD).
+
+## Building manual
+
+This section presents the crucial steps required to build the Dasharo Heads
+firmware. For more information, you may also refer to the official
+[Heads building documentation](https://osresearch.net/general-building/).
+
+### Requirements
+
+This guide was verified on Ubuntu 22.04. In practice, any Linux distribution
+with [Docker](https://www.docker.com/) support should be enough to complete it.
+
+Make sure that you have following packages installed:
+
+* Docker
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git
+
+    ```bash
+    sudo apt -y install git
+    ```
+
+* 1. Clone Dasharo Heads repository
+
+   ```bash
+    git clone https://github.com/Dasharo/heads.git
+   ```
+
+1. Navigate to the source code directory and checkout to the desired revision:
+
+    ```bash
+    cd heads
+    git checkout msi_ms7d25_v0.9.0
+    ```
+
+1. Start docker container:
+
+    ```bash
+    docker run --rm -it -v $PWD:$PWD -w $PWD \
+      3mdeb/heads-docker:3.0.0 /bin/bash
+    ```
+
+1. Inside of the container, start the build process:
+
+=== "PRO Z690-A (WIFI) DDR4"
+
+    ```bash
+    BOARD=msi_z690a_ddr4 make
+    ```
+
+    This will produce a Dasharo binary placed in
+    `build/x86/msi_z690a_ddr4/dasharo-msi_z690a_ddr4-*.rom`.
+
+=== "PRO Z690-A (WIFI)"
+
+    ```bash
+    BOARD=msi_z690a_ddr5 make
+    ```
+
+    This will produce a Dasharo binary placed in
+    `build/x86/msi_z690a_ddr5/dasharo-msi_z690a_ddr5-*.rom`.
+
+## Transition from Dasharo UEFI to Heads variant
+
+**Recommended**
+
+Please use [Dasharo Tools Suite](../../dasharo-tools-suite/overview.md) with
+Heads subscription credentials to transition to Heads.
+
+**Manual**
```

Comment:
Let's recommend switching back to AMI firmware before using FlashBIOS with the heads binary and povide explanations or include links to instructions on how to perform these steps.

Consider incorporating admonitions from https://squidfunk.github.io/mkdocs-material/reference/admonitions/ to grab the user's attention, similar to what we did here: https://docs.dasharo.com/variants/novacustom_nv4x_adl/heads/#switching-from-dasharo-uefi-to-heads

Reply:
https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-325e58bb2b51acd669b828c9d7e5f51ebc864ab5da29674d065c031e7c420553R109

https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-325e58bb2b51acd669b828c9d7e5f51ebc864ab5da29674d065c031e7c420553R172

Reply:
https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-66eaec24cfbda8a20d1ec829dfe08040814f5bd46b20274382f9092a3d784e7bR172-R177
https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-66eaec24cfbda8a20d1ec829dfe08040814f5bd46b20274382f9092a3d784e7bR161-R166

Reply:
great, resolving